staking/integration-tests/src/integrity_pool/helper_functions.rs

```diff
@@ -7,26 +7,28 @@ use {
         airdrop_spl,
         initialize_ata,
     },
-    anchor_lang::solana_program::pubkey::Pubkey,
     integrity_pool::utils::types::FRAC_64_MULTIPLIER,
-    solana_sdk::signature::Keypair,
+    solana_sdk::{
+        signature::Keypair,
+        signer::Signer,
+    },
 };
 
 
 pub fn initialize_pool_reward_custody(
     svm: &mut litesvm::LiteSVM,
     payer: &Keypair,
-    pyth_token_mint: Pubkey,
```

Comment:
mmmm seems like you can keep Pubkey here?

Reply:
airdrop_spl needs Keypair